packages/central-server/config/test.json5

```diff
@@ -2,6 +2,9 @@
 {
   "db": {
     "name": "tamanu-central-test",
+    "pool": {
+      "acquire": 5000,
```

Comment:
Added this so that the new test finishes in under 10s, hoping it doesn't break other tests (tho thats a bit of a smell if it does).